server/src/main/java/com/witness/server/service/impl/WorkoutLogServiceImpl.java

```diff
@@ -439,7 +439,7 @@ private Map<Long, Integer> getSetLogPositions(List<SetLog> setLogs) {
   private Map<Long, Integer> simplifyPositionSpecification(Map<Long, Integer> positions) {
     var sortedEntries = positions.entrySet().stream()
         .sorted(Map.Entry.comparingByValue())
-        .collect(Collectors.toList());
+        .toList();
```

Comment:
In [this comment](https://github.com/witness-org/witness/pull/109#issuecomment-1368563816), you said you wanted to go back to using `Stream.collect(Collectors.toList())` instead of `Stream.toList()`. I thought you reverted the respective changes in a2989598becef8dae634ad80ce076930178026bd? Is this `toList()` call here on purpose or did you forget to revert it?